integrations/fastembed/src/haystack_integrations/components/rankers/fastembed/ranker.py

```diff
@@ -0,0 +1,194 @@
+from typing import Any, Dict, List, Optional
+
+from haystack import Document, component, default_from_dict, default_to_dict, logging
+
+from .ranker_backend.fastembed_backend import _FastembedRankerBackendFactory
+
+logger = logging.getLogger(__name__)
+
+
+@component
+class FastembedRanker:
+    """
+    Ranks Documents based on their similarity to the query using [Fastembed models](https://qdrant.github.io/fastembed/examples/Supported_Models/).
+
+    Documents are indexed from most to least semantically relevant to the query.
+
+    Usage example:
+    ```python
+    from haystack import Document
+    from haystack_integrations.components.rankers.fastembed import FastembedRanker
+
+    ranker = FastembedRanker(model_name="Xenova/ms-marco-MiniLM-L-6-v2", top_k=2)
+
+    docs = [Document(content="Paris"), Document(content="Berlin")]
+    query = "What is the capital of germany?"
+    output = ranker.run(query=query, documents=docs)
+    docs = output["documents"]
```

Comment:
```suggestion
   print(output["documents"][0].content)
   
   # Berlin
```

I think adding an example of the output can help.